quality/fca/cypress/support/api/steps/api-response-steps.ts

```diff
@@ -116,14 +116,17 @@ Then('le corps de la réponse contient un JWT', function () {
     });
 });
 
-Then('le corps de la réponse contient une erreur', function () {
-  cy.get('@apiResponse')
-    .its('body')
-    .then(async (body) => {
-      const errors = await validateDto(body, ApiErrorDto, {
-        forbidNonWhitelisted: true,
-        whitelist: true,
+Then(
+  'le corps de la réponse contient une erreur avec les champs error et error_description',
+  function () {
+    cy.get('@apiResponse')
+      .its('body')
+      .then(async (body) => {
+        const errors = await validateDto(body, ApiErrorDto, {
+          forbidNonWhitelisted: true,
+          whitelist: true,
+        });
+        expect(errors, JSON.stringify(errors)).to.have.length(0);
```

Comment:
Suggestion: actually the use of class validator seems over-engineered to me.

I suggest you just check that erro and error description are defined and delete ApiErrorDto.

Reply:
OK, ApiErrorDto and this call to validateDto are no more !